### ISSUE TITLE
Description of the Media Use options

### ISSUE THREAD

**Issue by kanasznagyzoltan** [Islandora-CLAW/CLAW#993]
Hello Developers!

Could you give me some info about what will be the usage of these options in the released software?
![image](https://user-images.githubusercontent.com/32355628/50284695-24a6f500-045a-11e9-8886-080ee966144a.png)

Thank you!

**Comment by dannylamb** [comment]
@kanasznagyzoltan If you're asking which are directly tied to functionality in the software, so far it's only Original File (default for triggering derivatives -- this is configurable, previously we used Preservation Master File), with Service File and Thumbnail Image used for display.

The rest are from PCDM as @mjordan mentioned and we have analogs for them in Islandora 7.x, but no functionality is directly tied to them yet in Islandora CLAW.  It's safe to assume at some point we'll be basing logic around them, but none of that will be included in the initial release.  For example, indexing Transcripts and Extracted Text in Solr seems like an obvious thing to do.  But for now, they're just labels.

**Comment by rosiel** [comment]
@ajstanley and I were discussing today how from a preservation perspective, we'll want to run things like reports and checksums on "Preservation Master"'s. In many cases this is the same as Original File though not always. Is there a strict necessity for only allowing one pdcm:use?

**Comment by whikloj** [comment]
Is this a question of which tags an individual institution might prefer to use? It seems to me that while I understand Original File, I would (in concert with the Archivists) agree to use it or Preservation Master and change the Contexts to use one or the other.

**Comment by rosiel** [comment]
No, i mean that when you're thinking "i should make derivatives, what should i look for as a source?" then Original File makes sense. When you're thinking "I really care about the archival file; which one is it" then Preservation Master makes sense. As far as I see, _we are no longer making the assumption that the "OBJ" equivalent is both archival and original._ This is good. Sometimes it makes sense to export from a proprietary original format to something more open/preservation friendly. But... in 90% of our cases they *are* the same thing. So whereas we have to hard-code some taxonomy term into the dang functions, and the taxonomy terms are now (unlike 'OBJ') meaningful outside of the islandora community, why have we placed a seemingly arbitrary limitation that Media may only have one PCDM use type?

**Comment by dannylamb** [comment]
@rosiel It's because we're using the Media Use vocabulary like a DSID in 7.x, and so that assumption just kinda carried over.  But I don't think there's anything limiting you other than the field settings themselves  E.g. make `field_media_use` multivalued and you should be okay.  There's no situation that I can think of in the logic that would get messed up by that.

So just try it, maybe.  Tagging something as Orignal File and Preservation Master File when the single file serves both purposes _should_ sort itself out.

**Comment by dmoses** [comment]
One of the items we have on our "to do" list is format normalization. To provide an example, our users will upload a PDF (Original File), but for preservation we may want to convert that original file to the preferred preservation format PDF/a (Preservation Master).

@dannylamb we have a use case that needs to utilize multiple media use terms to trigger microservices. When we upload a PDF (Original File), we want to extract the textual content of the PDF for indexing (Extracted Text).

**Comment by elizoller** [comment]
I think the multiple media use terms issue is solved by https://github.com/Islandora-CLAW/CLAW/issues/1142?

**Comment by seth-shaw-unlv** [comment]
Yes, @elizoller, that PR did enable us to apply multiple use terms to a single media.

Although I think @dmoses' use case is the ability to trigger multiple actions based on a single use term. This is also currently possible with (possibly multiple) Contexts. E.g. you could edit the existing pdf_original_file Context to trigger a preservation master derivative and an extract text action in addition to the existing thumbnail derivative.

Alternatively, if you wanted to make this a bit more complex, once we get Crayfits included (or if you have some other validation mechanism), you could make a Context that fires only if it is not a PDF/A.
